contrib/util/language_translations/buildLanguageDatabase.pl

```diff
@@ -553,7 +553,7 @@ ()
   `definition` mediumtext,
   UNIQUE KEY `def_id` (`def_id`),
   KEY `cons_id` (`cons_id`) 
-) ENGINE=MyISAM AUTO_INCREMENT=".$tempCounter." ;
+) ENGINE=InnoDB AUTO_INCREMENT=".$tempCounter." ;
 \n
```

Comment:
oh, forgot about the language tables. To make this persistent, need to change it in the script that builds the language tables:
https://github.com/openemr/openemr/blob/master/contrib/util/language_translations/buildLanguageDatabase.pl
lines 422, 481,  556